lib/livebook_web/live/home_live.ex

```diff
@@ -157,18 +160,40 @@ defmodule LivebookWeb.HomeLive do
     end
   end
 
+  defp update_notification(%{version: nil} = assigns), do: ~H""
+
+  defp update_notification(assigns) do
+    ~H"""
+    <div class="px-2 py-2 bg-blue-200 text-gray-900 text-sm text-center">
+      <span>
+        Livebook v<%= @version %> available! Check out the news on
+        <a class="font-medium border-b border-gray-900 hover:border-transparent" href="https://livebook.dev/" target="_blank">
+          livebook.dev
+        </a>
+        <%= if @instructions_url do %>
+          and follow the
+          <a class="font-medium border-b border-gray-900 hover:border-transparent" href={@instructions_url} target="_blank">
+            update instructions
+          </a>
+        <% end %>
+        🚀
+      </span>
+    </div>
+    """
+  end
+
   defp memory_notification(assigns) do
     ~H"""
     <%= if @app_service_url && @memory.free < 30_000_000 do %>
-      <div class="flex justify-between items-center border-b border-gray-200 pb-4 text-gray-700">
-        <span class="flex items-end">
-          <.remix_icon icon="alarm-warning-line" class="text-xl mr-2" />
-          <span>
-          Less than 30 MB of memory left, consider adding more resources to
-          <a class="font-semibold" href={@app_service_url} target="_blank">the instance</a>
-          or closing <a class="font-semibold" href="#running-sessions">running sessions</a>.
-          </span>
-        </span>
+      <div class="px-2 py-2 bg-red-200 text-gray-900 text-sm text-center">
+        Less than 30 MB of memory left, consider adding more resources to
```

Comment:
You don't want to keep the icon? :D It looked cool. Your call!

Reply:
![image](https://user-images.githubusercontent.com/17034772/163692994-84d43fa6-69b6-49f2-9938-7daf6cb17338.png)